ui/components/ui/disclosure/disclosure.scss

```diff
@@ -23,4 +24,31 @@
       font-size: 12px;
     }
```

Comment:
Could we replace these static font sizes with a `Text` component?

Reply:
@georgewrmarshall 

hmm this one might be a little tricky since the content may be elements that are not compatible with the text variants. e.g. It may be a `<div>` which is incompatible as a child of `<p>`

the way I was going about it for the new SecurityProviderBannerAlert was  passing the Text elements as children

ref: https://github.com/MetaMask/metamask-extension/pull/19825/files
see `ui/components/app/security-provider-banner-alert/security-provider-banner-alert.js`

Reply:
I see. You could also change the root HTML element that is being rendered using the `<Text as="div"` prop. That way the typographic styles are attached using the className on the Text component but the underlying element is a div.

Reply:
I like this suggestion. Ty! Will add to another PR shortly 👍